src/style.css

```diff
@@ -1 +1,71 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-weight: 500;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header_logo {
+  display: flex;
+  align-items: center;
+}
+
+.navigation {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  flex-shrink: 0;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.navigation__li {
+  list-style: none;
+  padding: 0.3px 0;
+}
+
+.navigation__link {
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  color: #000;
+  position: relative ;
+  display: inline-block;
+  padding: 22.5px 0;
+}
+
+.navigation__link:hover {
+  color: rgba(0, 172, 220, 1);
+}
+
+.navigation :not(:first-child) {
```

Comment:
```suggestion
.navigation:not(:first-child) {
```

Reply:
But my code doesn't work when I've made that changes(